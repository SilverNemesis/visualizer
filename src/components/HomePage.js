import React from 'react'

const HomePage = () => {
  const code = `<Section outer>
  <Section>
    <Navigation />
  </Section>
  <Section grow center>
    <Route exact path="/" component={Home} />
    <Route path="/sort" component={Sort} />
  </Section>
</Section>`
  return (
    <pre style={{ color: 'black' }}><code>{code}</code></pre>
  )
}

export default HomePage