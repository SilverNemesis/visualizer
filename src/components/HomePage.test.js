import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'
import HomePage from './HomePage'

const renderer = new ShallowRenderer();

it('renders correctly', () => {
    renderer.render(<HomePage />)
    const tree = renderer.getRenderOutput()
    expect(tree).toMatchSnapshot()
});