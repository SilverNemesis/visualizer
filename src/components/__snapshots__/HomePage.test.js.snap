// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<pre
  style={
    Object {
      "color": "white",
    }
  }
>
  <code>
    &lt;Section outer&gt;
  &lt;Section&gt;
    &lt;Navigation /&gt;
  &lt;/Section&gt;
  &lt;Section grow center&gt;
    &lt;Route exact path="/" component={Home} /&gt;
    &lt;Route path="/sort" component={Sort} /&gt;
    &lt;Route path="/maze" component={MazePage} /&gt;
  &lt;/Section&gt;
&lt;/Section&gt;
  </code>
</pre>
`;
